tests/models/test_chat_templates.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from guidance._chat import load_template_class, CHAT_TEMPLATE_CACHE
+import transformers
+
+def test_popular_models_in_cache():
+    # This test simply checks to make sure the chat_templates haven't changed, and that they're still in our cache.
+    # If this fails, the models have had their templates updated, and we need to fix the cache manually.
+
+    models_to_test = [
+        "microsoft/Phi-3-mini-4k-instruct", # Phi-3
+        "meta-llama/Meta-Llama-3-8B-Instruct", # Llama-3
+        "meta-llama/Llama-2-7b-chat-hf", # Llama-2
+        "mistralai/Mistral-7B-Instruct-v0.2", # Mistral-7B-Instruct-v0.2
+    ]
+    for model_id in models_to_test:
```

Comment:
Please do this via `pytest.parametrize` not a loop inside the test.

Reply:
done